src/codegen/codegen_neuron_cpp_visitor.hpp

```diff
@@ -130,7 +130,6 @@ class CodegenNeuronCppVisitor: public CodegenCppVisitor {
     /*                     Common helper routines accross codegen functions                 */
     /****************************************************************************************/
 
-
```

Comment:
Nitpick: just whitespace change, can be reverted (unless formatter complains).

Reply:
similarly, it can stay, because the formatter allows it :)